viewer/packages/pointclouds/src/CognitePointCloudModel.ts

```diff
@@ -370,7 +436,9 @@ export class CognitePointCloudModel {
    */
   traverseStylableObjects(callback: (annotationMetadata: PointCloudObjectMetadata) => void): void {
     for (const obj of this.pointCloudNode.stylableObjectAnnotationMetadata) {
-      callback(obj);
+      if ('annotationId' in obj) {
+        callback(obj as PointCloudObjectMetadata);
```

Comment:
Did we have e.g. a "`isAnnotationPointCloudVolume`" for this? Should we have one?